File: programmeren1/extra/oefeningen/week5/tekenrechthoek.py
'''
Week 5 - Extra Oefeningen
Oefening 7
'''

def main():
    """Verkrijg hoogte en breedte en teken een rechthoek"""

    MAX = 20
    
    hoogte = int(input("Geef de hoogte in: "))
    breedte = int(input("Geef de breedte in: "))

    hoogte = MAX if hoogte > MAX else hoogte
    breedte = MAX if breedte > MAX else breedte
    
    for _ in range(hoogte):
        print("*" * breedte)

    # interesting, but bad style. will return list of None
    #[print("*" * breedte) for _ in range(hoogte)]
        
if __name__ == "__main__":
    main()

File: programmeren1/extra/oefeningen/week5/TekenRechthoek.java
/*
Week 5 - Extra Oefeningen
Oefening 7
*/
import java.util.Scanner;

public class TekenRechthoek {

    public static void main(String[] args) {

        Scanner reader = new Scanner(System.in);
        System.out.print("Geef de hoogte in: ");
        int hoogte = Integer.parseInt(reader.nextLine());
        System.out.print("Geef de breedte in: ");
        int breedte = Integer.parseInt(reader.nextLine());

        do {
            for (int i = 0; i < breedte; i++)
                System.out.print('*');
            System.out.println();
        }
        while (--hoogte > 0);
    }
}
